packages/nimble-components/src/wafer-map/modules/zoom-handler.ts

```diff
@@ -0,0 +1,126 @@
+import { select } from 'd3-selection';
+import {
+    zoom,
+    ZoomBehavior,
+    zoomIdentity,
+    ZoomTransform,
+    zoomTransform
+} from 'd3-zoom';
+import type { DataManager } from './data-manager';
+import type { RenderingModule } from './rendering';
+
+/**
+ * ZoomHandler deals with user interactions and events like zooming
+ */
+export class ZoomHandler {
+    private zoomTransform: ZoomTransform = zoomIdentity;
+    private readonly minScale = 1.1;
+    private readonly minExtentPoint: [number, number] = [-100, -100];
+    private readonly extentPadding = 100;
+
+    public constructor(
+        private readonly canvas: HTMLCanvasElement,
+        private readonly zoomContainer: HTMLElement,
+        private readonly dataManager: DataManager,
+        private readonly renderingModule: RenderingModule
+    ) {}
+
+    public attachZoomBehavior(): void {
+        const zoomBehavior = this.createZoomBehavior();
+        zoomBehavior(select(this.canvas as Element));
+    }
+
+    private createZoomBehavior(): ZoomBehavior<Element, unknown> {
+        this.canvas.addEventListener('wheel', event => event.preventDefault());
```

Comment:
Chrome devtools is complaining about not configuring the passive property:
![image](https://user-images.githubusercontent.com/1588923/210456646-fa30325d-a5cb-44f0-8591-35b39d9c5293.png)

In this case we explicitly do not want it to be passive, so we should set passive: false as shown in the [d3 issue](https://github.com/d3/d3-zoom/issues/231#issuecomment-802713799).

```suggestion
        this.canvas.addEventListener('wheel', event => event.preventDefault(), {passive: false});
```